statsmodels/discrete/discrete_model.py

```diff
@@ -3476,13 +3477,13 @@ def llnull(self):
 
         return res_null.llf
 
-    @cache_readonly
-    def fittedvalues(self):
-        return np.dot(self.model.exog, self.params[:self.model.exog.shape[1]])
+    @property
+    def resid(self):
+        return self.resid_response
```

Comment:
What is the logic that determines when `resid` should be an alias for `resid_response`?  Could go in a docstring?

Reply:
Only GLM and discrete models have `resid_response`, because Skipper thought it was ambiguous what `resid` without qualifier would be.
Now. the definition is that resid without qualifier is the same as resid_response.

Reply:
Thanks; putting that in a docstring would be helpful.